doc/examples/svg/lib/main.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flame/game.dart';
+import 'package:flame/svg.dart';
+import 'package:flame/components/component.dart' show SvgComponent;
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyGame());
+
+class MyGame extends BaseGame {
+
+  SvgComponent android = null;
```

Comment:
Don't need to initialize with null, that's totally redundant.

Reply:
Okay, javascript habits...

Reply:
hahaha such a great thought to have null and undefined! :P